js_modules/dagit/packages/core/.storybook/preview.js

```diff
@@ -7,39 +7,77 @@ import '@blueprintjs/popover2/lib/css/blueprint-popover2.css';
 import {FontFamily} from '../src/ui/styles';
 import {GlobalDialogStyle} from '../src/ui/Dialog';
 import {GlobalPopoverStyle} from '../src/ui/Popover';
+import {GlobalSuggestStyle} from '../src/ui/Suggest';
+import {GlobalToasterStyle} from '../src/ui/Toaster';
+import {GlobalTooltipStyle} from '../src/ui/Tooltip';
+import {ColorsWIP} from '../src/ui/Colors';
 
 import * as React from 'react';
 import {MemoryRouter} from 'react-router-dom';
 
 import {createGlobalStyle} from 'styled-components/macro';
 
-
 const GlobalStyle = createGlobalStyle`
   * {
     box-sizing: border-box;
+  }
+
+  html, body {
+    color: ${ColorsWIP.Gray800};
     -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
   }
 
-  body, input, textarea, button, select {
-    font-family: ${FontFamily.default};
+  a,
+  a:hover,
+  a:active {
+    color: ${ColorsWIP.Link};
   }
 
-  body ul, body li {
+  body {
     margin: 0;
+    padding: 0;
+  }
+
+  body, input, select, textarea {
+    font-family: ${FontFamily.default};
+  }
+
+  button {
+    font-family: inherit;
+  }
+
+  code, pre {
+    font-family: ${FontFamily.monospace};
+    font-size: 16px;
   }
 
   .material-icons {
     display: block;
   }
+
+  /* todo dish: Remove these when we have buttons updated. */
+
+  .bp3-button .material-icons {
+    position: relative;
+    top: 1px;
+  }
+
+  .bp3-button:disabled .material-icons {
+    color: ${ColorsWIP.Gray300}
+  }
```

Comment:
I also pulled a bunch of our more recent Dagit UI base changes into Storybook, since it was lacking.